changelog/fragments/2879.yaml

```diff
@@ -0,0 +1,12 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      bundle create: fix overwrite existing bundle bug
+
+      https://github.com/operator-framework/operator-sdk/issues/2879
```

Comment:
We do not need add the PR link any more. Could you please remove?

Reply:
This is a link to the issue itself for more context.

Reply:
It will be added automatically because this we do not need to add it anymore.

Reply:
The pull request link will be added, not the issue.